roles/testnode/vars/redhat_6.yml

```diff
@@ -8,6 +8,12 @@ common_yum_repos:
     enabled: 1
     gpgcheck: 0
     priority: 2
+  centos6-fcgi-ceph:
+    name: "Cent OS 6 Local fastcgi Repo"
+    baseurl: http://gitbuilder.ceph.com/mod_fastcgi-rpm-rhel6-x86_64-basic/ref/master/
```

Comment:
You should use the var ``gitbuilder_host`` here instead of hardcoding gitbuilder.ceph.com